src/actions/authAction.js

```diff
@@ -1,14 +1,18 @@
 class AuthAction {
     constructor(){
-        this.authenticated = false
+
+        this.authenticated = localStorage.hasOwnProperty("user")
     }
 
     login(cb){
+        console.log("In authActions!!")
         if(localStorage.hasOwnProperty("user")){
```

Comment:
Never use localStorage direct. Instead of it make a global useable function for local storage and always import that where you need to use.